### ISSUE TITLE
Best practice for finetuning pretrained Faster-RCNN model

### ISSUE THREAD

**Issue by seawee1** [rbgirshick/py-faster-rcnn#914]
I trained my detector on a specific dataset where it is performing pretty well. Now I want to finetune the model to another image domain, object classes stay the same. The new domains dataset is relatively small. I tried reducing the learning rate by a factor of 10 but training overfitts pretty fast after only maybe 10k iterations.

Are there any best practices to finetune a Faster RCNN model. Are there any layers (besides the first few backbone CNN layers) worth freezing? (However, are there that many other layers to freeze besides the backbone? If I understood correctly the detection architectur only uses a small number of FC and Conv layers for bounding box regression and region classification).
Any tips would be much appreciated! :)
